specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/consumption.json

```diff
@@ -1868,6 +1907,43 @@
         }
       }
     },
+    "Tags": {
+      "description": "A resource listing all tags.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/TagProperties",
+          "title": "Tag properties"
+        }
+      }
+    },
+    "TagProperties": {
+      "description": "The properties of the tag.",
+      "properties": {
+        "tags": {
+          "description": "Tags.",
```

Comment:
One more thing : Please avoid using property name as the description. Try to add a little more info :)

Reply:
Sure! Will keep this in mind for future as well.